### PR TITLE
[doc] NetexIDF: fixed calendar specs

### PR DESCRIPTION
**Pull request by datanel**

### CODE REVIEW

File: src/documentation/netex_idfm_to_ntfs.md [hunk 1]
```diff
@@ -325,15 +325,22 @@ The `service_id` property of the calendar in the NTFS is specified by an auto-in
 
 Active dates of the calendar are specified by:
 - `DayType` nodes describing the active days of a week
-  + days of the week are listed in `DayType/properties[]/PropertyOfDay/DaysOfWeek nodes`. 
-  + Expected values MUST be one of `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`.
+  + days of the week are listed in `DayType/properties[]/PropertyOfDay/DaysOfWeek` nodes.
```
Comment (prhod) [inline src/documentation/netex_idfm_to_ntfs.md]:
```suggestion
  + days of the week are listed in `DayType/properties/PropertyOfDay[]/DaysOfWeek` nodes. 
```

File: src/documentation/netex_idfm_to_ntfs.md [hunk 2]
```diff
@@ -325,15 +325,22 @@ The `service_id` property of the calendar in the NTFS is specified by an auto-in
 
 Active dates of the calendar are specified by:
 - `DayType` nodes describing the active days of a week
-  + days of the week are listed in `DayType/properties[]/PropertyOfDay/DaysOfWeek nodes`. 
-  + Expected values MUST be one of `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`.
+  + days of the week are listed in `DayType/properties[]/PropertyOfDay/DaysOfWeek` nodes. 
+    + Expected values MUST be one of `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`.
+  + `DayType/properties` nodes are optionals. You can have a `DayType` node without active days of a week.
```
Comment (prhod) [inline src/documentation/netex_idfm_to_ntfs.md]:
```suggestion
  + `DayType/properties` node is optional, a `DayType` node may not have any active days of a week.
```

**Comment by datanel** [comment]
@prhod done

File: src/documentation/netex_idfm_to_ntfs.md [hunk 3]
```diff
@@ -325,15 +325,22 @@ The `service_id` property of the calendar in the NTFS is specified by an auto-in
...
 - `DayTypeAssignment` nodes with 2 possible uses (they are applied in the order they appear in the file):
   + Activate or deactivate a specific day on a DayType (with the nodes `IsAvailable` and `Date`)
+    + the node `IsAvailable` is optional. If not present the default value is `true`.
   + Apply the active days of the referenced DayType on an `OperatingPeriod`
 
 All resulting calendars are to be restricted between `ValidBetween/FromDate` and `ValidBetween/ToDate` specified at the top level of the file in `GeneralFrame`.
 
+Active dates are read in the following order:
+- dates from operating periods
+- dates added (`DayTypeAssignment/IsAvailble` = true)
```
Comment (woshilapin) [inline src/documentation/netex_idfm_to_ntfs.md]:
```suggestion
- dates added (`DayTypeAssignment/IsAvailable` = true)
```

File: src/documentation/netex_idfm_to_ntfs.md [hunk 4]
```diff
@@ -325,15 +325,22 @@ The `service_id` property of the calendar in the NTFS is specified by an auto-in
...
+- dates removed (`DayTypeAssignment/IsAvailble` = false)
```
Comment (woshilapin) [inline src/documentation/netex_idfm_to_ntfs.md]:
```suggestion
- dates removed (`DayTypeAssignment/IsAvailable` = false)
```

**Pull request merged**